packages/odyssey-react/package.json

```diff
@@ -25,6 +25,7 @@
     "@svgr/cli": "^5.5.0",
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.2",
```

Comment:
yay! Don't we have a few tests already for hooks that are using the approach we put together ourselves when we paired a month or so ago? Would be great to refactor those in a subsequent PR so tests have a single voice.